src/posts/2021-06-14-Everyday_Accessibility.md

```diff
@@ -0,0 +1,121 @@
+	<img alt="The Check Accessibility menu option is activated on a simple PowerPoint deck where one slide is missing a title. The Accessibility pane shows the 'Missing slide title' error and the mouse hover text for that slide also reads '[No Title]'." src="/img/posts/2021-06-14-Everyday_Accessibility/PPT_A11yChecker.png" />
+	<figcaption>Microsoft's "Check Accessibility" feature helps ensure your document is inclusive.</figcaption>
+</figure>
+
+## Audio and Video Content
+
+### Have clear audio
+Though all of our devices can record audio and video on-demand, sound quality can vary.  Be sure to speak as clearly as possible, preferably at a steady pace and reasonable volume, and remove or avoid background noise including echo, empty room effect, humming, or white noise.
+
+> Using a microphone (even a common corded headset) can help amplify voices and minimize distracting unwanted sounds.
+
+### Reduce background music
+Instrumental music can enhance content’s quality, but not when it reduces the ability to hear the main speaker. When using background music, keep it to a minimum (such as during an intro) or, if used throughout, ensure it is very low.
+
+> Consider putting your background music at the volume of a whisper.
+
+### Provide text with multimedia
+As with images, video and audio can be engaging but also inaccessible without a text equivalent for people who are deaf-blind, having sensory processing disorders, or have hearing impairments. Having media content written out as text is vital for inclusiveness. Text equivalents can include transcripts that type out spoken dialogue, or an article that accompanies a video (as in a video that supplements a news article).
+
+> Where there’s media, make sure there’s also text.
+
+### Do not have any flashing content
+Content that flashes on the screen more than 3 times per second is dangerous as it can trigger a seizure in some viewers. Use rapid movement sparingly in your videos and ensure nothing flashes.
+
+> The Trace Center’s [Photosensitive Epilepsy Analysis Tool (PEAT)](https://trace.umd.edu/peat/) is a free, downloadable resource to identify seizure risks in  web content and software.
+
+## Advocate for Accessibility
+
+### Seek accessibility features
+Actively search for “accessibility” in the tools you use every day — from survey creation sites to website design platforms. Understand what you can do to make sure your output is inclusive.
+
+<figure role="figure" aria-label="Search for accessibility tips in your favorite systems like this Typeform How To article.">
+	<img alt="Help Center page from Typeform, an online survey creation service, regarding how to make accessible typeforms." src="/img/posts/2021-06-14-Everyday_Accessibility/Accessible_Typeforms.png" />
+	<figcaption>Search for accessibility tips in your favorite systems like this Typeform "How To" article.</figcaption>
+</figure>
```

Comment:
Can we use an example other than Typeform? They have poor AT support and have publicly stated that improving the accessibility of their platform isn't a priority.

Reply:
[Curse word] Figured as much. Yup, I'll take a gander now...